homework/week-2/jorgereina1986/Horoscope/app/src/main/res/layout/activity_virgo.xml

```diff
@@ -0,0 +1,33 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#ff274756"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="jorgereina1986.c4q.nyc.horoscope.SignDescription">
+
+    <TextView
+        android:id="@+id/signName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fontFamily="times"
```

Comment:
Since these style attributes are the same for each TextView every time, you can save these attributes as a style in your styles.xml. You add attributes like fontFamily, textColor, textSize, etc. one time and apply to your TextView by assigning that style (ex:style="@style/signName"). You will also be able to make changes to this style once and have them apply globally, rather than changing each attribute twelve times every time.